server/views.py

```diff
@@ -33,6 +34,29 @@ def index(request):
     }, request))
 
 
+def user(request, name=None):
+    template = loader.get_template('user.html')
+    user_info = get_user_info_dict(request.user)
+    user = User.objects.get(username=name if name else user_info['name'])
+
+    this_user = {}
+    this_user['full_name'] = '{} {}'.format(user.first_name, user.last_name)
+    this_user['avatar'] = user.avatar
+    this_user['name'] = user.username
+    return HttpResponse(template.render({
+        'user_info': json.dumps(user_info),
+        'this_user': json.dumps(this_user),
+        'notebook_list': json.dumps(
+            [{
+                'id': v[0],
+                'title': v[1],
+                'last_revision': NotebookRevision.objects
+                .filter(notebook_id=v[0]).last().created.isoformat(sep=' ')
```

Comment:
This is going to be pretty slow if we had a bunch of notebooks, we would probably do better to add a field in the notebook model itself to track this.

Reply:
Very good point, and also agreed we can handle this in a followup. I'll file an issue so we can keep track.